contracts/ForkingManager.sol

```diff
@@ -102,38 +102,32 @@ contract ForkingManager is IForkingManager, ForkableStructure {
     /**
      * @dev function that executes a fork proposal
      */
-    function executeFork() external onlyBeforeForking {
+    function executeFork1() external onlyBeforeForking {
```

Comment:
Add a comment: its imperative that this does not revert. That means that newImplementations need to contain proper addresses. I will make a separate PR to ensure that